main/webapp/modules/core/scripts/util/csrf.js

```diff
@@ -1,16 +1,29 @@
-CSRFUtil = {};
+CSRFUtil = {
+    _tokenMaxAge: 1800000 // half an hour in milliseconds (tokens live for 1 hour in the backend)
```

Comment:
Do you mean https://jwt.io/? So turning tokens into a base64 encoding of a JSON object with the actual token and a date? It sounds rather complex, but it could be done for sure… I was more thinking about adding the expiry date in an HTTP response header of the get-csrf-token command, it felt simpler, but still not really clear it was worth the trouble

Reply:
Yes, that's the correct JWT, but since the token is just a string currently, not a JSON object, it's perhaps not worth the hassle.